test_runner/fixtures/zenith_fixtures.py

```diff
@@ -1099,9 +1104,9 @@ def check_restored_datadir_content(zenith_cli: ZenithCli, test_output_dir: str,
                                                 restored_dir_path,
                                                 pgdata_files,
                                                 shallow=False)
-    print('filecmp result mismatch and error lists:')
-    print(mismatch)
-    print(error)
+    log.info('filecmp result mismatch and error lists:')
+    log.info(mismatch)
+    log.info(error)
```

Comment:
I'd print this as a single message.

Reply:
I fixed this and looked at other prints, but couldn't find similar multiline messages anywhere else. There may be other places to optimize the logs, but it's not as obvious to improve them.

Anyway, all logs are captured and displayed in the summary (when running `pytest -rA`). In this summary logs from different tests are not mixed with each other, so concurrent tests don't really affect anything log-related in CI.